### DISCUSSION TITLE
Not really into coding., how do I get it to run?

### DISCUSSION THREAD

**Discussion by KingoArcher** [rdeepak2002/reddit-place-script-2022#252] [Q&A]
Ive followed the steps (Mac User).  changed everything to fit my needs, just need to get it to run.  How do I do that?

**Comment by thoroc** [comment]
Have you got Python3 installed? If that is the case, please follow the steps:

* Open a new terminal window to the root of the project
* Create new environment: `virtualenv .venv` (Install `virtualenv` if needed be `pip install virtualenv`)
* Activate the environment: `souce $PWD/.venv/bin/activate`
* Install the dependencies: `pip install -r requirements.txt`
* Call the script with python: `python main.py`

It should work

**Reply by KingoArcher** [reply]
<img width="560" alt="Screen Shot 2022-04-03 at 10 00 23 AM" src="https://user-images.githubusercontent.com/102909508/161431632-2780f244-8f76-4d7f-a98a-283665bf8c5d.png">

Hi, Mind Me for being dumb, But I opened the terminal window at the folder containing the project and simply ran virtualenv .venv and his this issue arise.  Any ideas?

**Reply by thoroc** [reply]
Apologies for the late reply. I should have mentioned that you "just" need to create a virtual environment with your chosen tool. I have used `virtualenv` so much that I have forgotten it isn't part of the buitin one. If you want to use it, please do the following:

```
pip install virtualenv
```
or
```
python -m pip install virtualenv
```

**Comment by PopcornPlant** [comment]
Download PyCharm
Open the folder of files you download from this github in PyCharm.
Press "Yes" to Requirements
Type "python main.py" in the terminal box (bottom of screen).

**Comment by KingoArcher** [comment]
<img width="726" alt="Screen Shot 2022-04-03 at 10 06 32 AM" src="https://user-images.githubusercontent.com/102909508/161431887-146086be-28d9-4bd5-bf57-fe8050c7267c.png">

**Comment by KingoArcher** [comment]
<img width="1055" alt="Screen Shot 2022-04-03 at 10 08 54 AM" src="https://user-images.githubusercontent.com/102909508/161431982-db4f4e9c-bdac-476e-a04b-640cb3d79cb7.png">
I attempted running the start.sh line

**Reply by Skyline-9** [reply]
Hey, can you `git pull` the latest version of the repo? I believe someone fixed that bug in a recent merge
